downloads.html

```diff
@@ -65,13 +65,13 @@
                     <p>Below you can find download links to the latest files.</p>
                     <ul>
                         <li>
-                            <a href="ftp://download.tuxfamily.org/opendungeons/0.6.0/opendungeons-0.6.0.tar.xz" target="_blank">OpenDungeons 0.6.0 (Source code)</a>
+                            <a href="ftp://download.tuxfamily.org/opendungeons/0.7.1/opendungeons-0.7.1.tar.xz" target="_blank">OpenDungeons 0.7.1 (Source code)</a>
                         </li>
                         <li>
-                            <a href="ftp://download.tuxfamily.org/opendungeons/0.6.0/OpenDungeons-0.6.0-Win32-MSVS2013.zip" target="_blank">OpenDungeons 0.6.0 (Windows 32-bit binary)</a>
+                            <a href="ftp://download.tuxfamily.org/opendungeons/0.7.1/OpenDungeons-0.7.1-Win32-MSVS2013.zip" target="_blank">OpenDungeons 0.7.1 (Windows 32-bit binary)</a>
```

Comment:
Note that I've added 0.7.1 binary into the 0.7.0 folder since it is the same version. Actually, I think we could have gone for a 0.7.X or 0.7 folder instead (but to avoid broken links, we can still use 0.7.0).
So, you should either change that or move the binary if you prefer to have one path per release.